docs/basic-features/data-fetching/get-server-side-props.md

```diff
@@ -45,6 +45,18 @@ It can be tempting to reach for an [API Route](/docs/api-routes/introduction.md)
 
 Take the following example. An API route is used to fetch some data from a CMS. That API route is then called directly from `getServerSideProps`. This produces an additional call, reducing performance. Instead, directly import the logic used inside your API Route into `getServerSideProps`. This could mean calling a CMS, database, or other API directly from inside `getServerSideProps`.
 
+### getServerSideProps with Edge API Routes
+
+`getServerSideProps` can be used with both Serverless and Edge Runtimes, and you can set props in both. However, currently in Edge Runtime, you do not have access to the response object. This means that you cannot — for example — add cookies in `getServerSideProps`. To have access to the response object, you should **continue to use the Serverless runtime**, which is the default runtime.
```

Comment:
Is it serverless runtime? I thought its node runtime. Serverless is the name for execution model that lets you run server code without hosting a server

Reply:
Good spot!